### ISSUE TITLE
Render error on Ubuntu Linux 20.04

### ISSUE THREAD

**Issue by Zonalds** [faressoft/terminalizer#157]
I would get this error when I try to render any recording <demo.yml>, it would return this error:

```
Error: 
  Error: [12665:0220/072245.249262:ERROR:buffer_manager.cc(488)] [.DisplayCompositor]GL ERROR :GL_INVALID_OPERATION : glBufferData: <- error from previous GL command

```

Any idea what I could be doing wrong?

**Comment by ghost91-** [comment]
Same here on Arch Linux with sway on wayland.

**Comment by aral** [comment]
Ditto on elementary OS 5.1.7 Hera x86_64 with Pantheon / Mutter(Gala).

**Comment by aral** [comment]
Update: both PRs #97 (with warning) and #154 (without warning) fix the issue for me. And the render is frightfully slow (compared to the blistering speed of https://github.com/asciinema/asciicast2gif).

**Comment by fubarhouse** [comment]
Any chance of this getting evaluated and/or merged.
Using Arch w/ x11 here, and it's a no-go - same error as described.

```
Error: 
  Error: [150111:0317/121422.967445:ERROR:buffer_manager.cc(488)] [.DisplayCompositor]GL ERROR :GL_INVALID_OPERATION : glBufferData: <- error from previous GL command

```

**Comment by faressoft** [comment]
The issue appears to have been resolved and released in the new version v0.10.0 by #97. If you believe that there is still something missing or unresolved, please feel free to reopen it.
